build/configs/artik053/tools/openocd/partition_gen.sh

```diff
@@ -0,0 +1,141 @@
+#!/bin/bash
+
+###########################################################################
+#
+# Copyright 2016-2017 Samsung Electronics All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+#
+###########################################################################
+
+# This script reads partition size and name list from .config and
+# creates a partition map cfg file for openocd.
+# Created partition map cfg file can be included in the main openocd cfg script
+# for flashing.
+
+source .config
+
+# Path ENV
+BOARD_NAME=${CONFIG_ARCH_BOARD}
+OS_DIR_PATH=${PWD}
+BUILD_DIR_PATH=${OS_DIR_PATH}/../build
+BOARD_DIR_PATH=${BUILD_DIR_PATH}/configs/${BOARD_NAME}
+OPENOCD_DIR_PATH=${BOARD_DIR_PATH}/tools/openocd
+
+# FLASH BASE ADDRESS (Can it be made to read dynamically from .config?)
```

Comment:
As you said, it sounds good to read it from .config.  May i know what stops now to read at runtime ?

Reply:
There is no config entry for this, so it has been hard coded here.